src/VisualStudio/Core/Test/Telemetry/VSTelemetryLoggerTest.vb

```diff
@@ -1,4 +1,5 @@
-﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿#If False Then ' not supported for now
```

Comment:
test is disabled since right now there is no way to do unit test with new API. in talk with telemetry team about this.